docs/workflow/README.md

```diff
@@ -46,33 +46,37 @@ You may need to build the tree in a combination of configurations. This section
 
 A quick reminder of some concepts -- see the [glossary](../project/glossary.md) for more on these:
 
-* **Debug configuration** -- Non-optimized code.  Asserts are enabled.
-* **Checked configuration** -- Optimized code. Asserts are enabled.  Only relevant to CoreCLR.
-* **Release configuration** -- Optimized code. Asserts are disabled. Runs at full speed, and suitable for performance profiling. Somewhat poorer debugging experience.
+* **Debug Configuration** -- Non-optimized code.  Asserts are enabled.
+* **Checked Configuration** -- Optimized code. Asserts are enabled.  Only relevant to CoreCLR runtime.
+* **Release Configuration** -- Optimized code. Asserts are disabled. Runs at best speed, and suitable for performance profiling. You will have limited debugging experience.
 
-When we talk about mixing configurations, we're discussing three sub-components:
+When we talk about mixing configurations, we're discussing following sub-components:
   
-* **CoreCLR** (often referred to as the runtime, most code under src/coreclr) -- this is the execution engine for managed code. It is written in C/C++. When built in a debug configuration, it is easier to debug into it, but it executes managed code more slowly - so slowly it will take a long time to run the managed code unit tests.
-* **CoreLib** (also known as System.Private.CoreLib - code under src/coreclr/System.Private.CoreLib) -- this is the lowest level managed library. It has a special relationship with the runtime -- it must be in the matching configuration, e.g., if the runtime you are using was built in a debug configuration, this must be in a debug configuration.
-* **All other libraries** (most code under src/libraries) -- the bulk of the libraries are oblivious to the configuration that CoreCLR/CoreLib were built in. Like most code they are most debuggable when built in a debug configuration, and, happily, they still run sufficiently fast in that configuration that it's acceptable for development work.
+* **Runtime** is the execution engine for managed code and there are two different runtimes available. Both runtimes are written in C/C++, therefore, easier to debug when built in a Debug configuration.
+    * CoreCLR VM is the primary desktop runtime which if built in Debug Configuration it executes managed code very slowly. For example it will take a long time to run the managed code unit tests. The code lives under [src/coreclr](../../src/coreclr).
```

Comment:
Nit: The previous paragraph talks about different runtimes, and this one switches to VM. We should be consistent.